great_expectations/execution_engine/execution_engine.py

```diff
@@ -619,7 +593,8 @@ def _split_table_metric_domain_kwargs(
                     map(lambda element: f'"{element}"', unexpected_keys)
                 )
                 logger.warning(
-                    f'Unexpected key(s) {unexpected_keys_str} found in domain_kwargs for domain type "{domain_type.value}".'
+                    f"""Unexpected key(s) {unexpected_keys_str} found in domain_kwargs for domain type \
```

Comment:
Is the `\` a stray?

Reply:
@billdirks I added it on purpose (to avoid `PyCharm` line length warnings).  Thanks.

Reply:
Fixed.